packages/patternfly-4/react-core/package.json

```diff
@@ -6,7 +6,7 @@
   "module": "dist/esm/index.js",
   "types": "dist/js/index.d.ts",
   "patternfly:src": "src/",
-  "sideEffects": false,
+  "sideEffects": ["*.css"],
```

Comment:
Is this something that should be done in core or something that should be done by the applications webpack?

Reply:
After a good bit of research over the past few days, I have determined that this should be done at the patternfly package level. Since the react-core package is NOT entirely pure ( the .css files have side effects via adding styles to the dom), the sideEffect property in the core package.json should reflect that. This way when a user just wants to import the base.css from core, they don't have to worry about the file being dropped unintentionally in production mode. 

See https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free

Reply:
@dlabaj Looks like Create-react-app handles this on the consumer application side. Here is a snippet from their production config:

            // Don't consider CSS imports dead code even if the
            // containing package claims to have no side effects.
            // Remove this when webpack adds a warning or an error for this.
            // See https://github.com/webpack/webpack/issues/6571
            sideEffects: true,

@seanforyou23 Maybe we should just handle it on the seed app side as a part of the end users webpack configuration?

Reply:
Thanks @ibolton336 I didn't realize about create-react-app.